### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by May0428** [aojeda/PEB#1]
Hello, I am a second-year graduate student of Chongqing University of Posts and Telecommunications，China. I want to use your tool kit because of the project needs, but I don't know how to use it. Could you please teach me how to use it?

**Comment by aojeda** [comment]
Hey there, that's an old repo, [DSI](https://github.com/aojeda/dsi) is what you really want.

**Comment by May0428** [comment]
Hello, do I just need to unpack the compressed package and rename it as DSI as you wrote in GitHub, put it under the Eeglab file, then I can use the function POP_RSBL, don't I need to carry out MEX from MATLAB to C?Thank you very much!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "aojeda/PEB"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月19日(星期一) 晚上10:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [aojeda/PEB] Question (#1)





 
Hey there, that's an old repo, DSI is what you really want.

**Comment by aojeda** [comment]
Yes, that should be it. There is only one mex file on the toolbox that should be auto-compiled for your machine upon first use (you don't need to worry about it, but for this to work you need to have Matlab properly configured with a compiler, see `mex setup`). The main algorithm can run without the mex file but it will be very very slow.

**Comment by May0428** [comment]
I'm sorry to trouble you again. I met this problem in the process of compiling. Could you please tell me how to solve it？Thank you very much! you are a nice guy!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "aojeda/PEB"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月19日(星期一) 晚上11:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [aojeda/PEB] Question (#1)





 
Yes, that should be it. There is only one mex file on the toolbox that should be auto-compiled for your machine upon first use (you don't need to worry about it, but for this to work you need to have Matlab properly configured with a compiler, see mex setup). The main algorithm can run without the mex file but it will be very very slow.
